README.md

```diff
@@ -1,59 +1,64 @@
-# Volograms Unity SDK 
+# Volograms Unity SDK
 
-# Prebuilt
+This is the [Volograms](https://www.volograms.com/) SDK for Unity, in the form of a plugin. You can use this plugin to create an app that plays volograms.
 
-Prebuilt versions of the Unity plugins are packaged into a `unitypackage` file in the [releases page](https://github.com/Volograms/volograms_unity_plugin/releases).
+## Supported Platforms 
 
-# Developer Guide
+* macOS - Excluding M1 Macs.
+* iOS
+* Windows
+* Android - Work-in-progress, expect issues!
 
-This repo contains the projects to build the native Volograms player plugins for Unity. It also uses code taken from [this repository](https://bitbucket.org/volograms/vol_av/src/master/). Each platform has a separate project and each project contains a README that documents how to build their respective plugin. When a plugin is built, it is copied to the `UnityVol` folder, which can be dragged and dropped into Unity. 
+Support is planned for M1 MacOS and GNU/Linux builds.
 
-## Platform Avaialble 
-- Mac (OSX, excluding M1 Macs)
-- iOS
-- Windows
-- Android (WIP, expect issues)
+## Getting Started
 
-## TODO Platforms 
-- M1 Macs
-- Linux
+* The easiest way to get started using is to download a pre-built plugin from the [Releases](https://github.com/Volograms/volograms_unity_plugin/releases) page.
+* If you would prefer to build the plugin from source, then you can clone this respository, and follow the [Developer Guide](#developer-guide).
 
-## Getting started
+## Unity Settings
+
+The libraries are exported into the [`VologramsToolkit`](./VologramsToolkit/Plugins/) folder. The `VologramsToolkit` folder can be imported directly into Unity. 
+
+After importing the plugin into Unity, ensure that the Unity settings for each plugin is correct. For example, we don't want to use the iOS plugin in a macOS build. 
```

Comment:
Typo/grammar: should be "ensure that the Unity settings... are correct" (not 'is')

Reply:
thanks, think i got it!